Add bank ac ui

AbhimanyuT0800 opened the pull request.

AbhimanyuT0800 committed:
core : add bank account details constants added

AbhimanyuT0800 committed:
features : add bank account details page addded

- add account constants added

AbhimanyuT0800 committed:
Merge branch 'main' of https://github.com/LX-PROJECTS/renthub into add-bank-ac

AbhimanyuT0800 committed:
features: add bank account details created

- add bank account consts added
- add bank account textfields created

AbhimanyuT0800 committed:
Merge branch 'main' of https://github.com/LX-PROJECTS/renthub into add-bank-ac

AbhimanyuT0800 committed:
back button name changed

AbhimanyuT0800 committed:
uncommented a line

Sireen-Official commented on lib/core/constants/Payment_constants/add_account_constants.dart:
Folder name must follow snake_case format

Sireen-Official commented on lib/core/constants/Payment_constants/add_account_constants.dart:
```diff
@@ -0,0 +1,25 @@
+part 'add_account_constants.g.dart';
+
+class AddBankAccountConstants {
+  final String textCancel = 'Cancel';
```
Prefix everything with txt to follow project standards

Sireen-Official commented on lib/features/payment/pages/add_bank_ac_details_page.dart:
```diff
@@ -0,0 +1,150 @@
+}
+
+// provider for controll dropdown bank names
+final StateProvider<int?> selectedBankIndexProvider = StateProvider<int?>(
```
Use hooks

AbhimanyuT0800 committed:
features: add bank account ui fixed

- constants name updated follows standerd
- hooks used instand of stateProvider
-

AbhimanyuT0800 committed:
folder name updated

AbhimanyuT0800 committed:
Changed folder name by disabling ignoreCase flag

AbhimanyuT0800 committed:
deleted payment constants

AbhimanyuT0800 committed:
added back payment constants
